bramkouwen/Kouwen_A_Rabari_A_AutomotiveMusicMixer
# Dev.ak.code

## What does the PR do?
* It updates js file location

## How do you check this PR?
* Pull down the branch
*  See js file location name change

## Screenshots 
None.
